src/jvmMain/kotlin/com/algolia/search/model/ClientDate.kt

```diff
@@ -10,10 +10,13 @@ import java.util.*
  * JVM implementation converting a [String] or a [Long] into a [Date] format. Relies on ISO8601.
  */
 @Serializable(KSerializerClientDate::class)
-public actual data class ClientDate actual constructor(override val raw: String) : Raw<String> {
+public actual data class ClientDate internal actual constructor(override val raw: String) : Raw<String> {
 
-    actual constructor(timestamp: Long) : this(DateISO8601.format(timestamp))
+    internal actual constructor(timestamp: Long) : this(DateISO8601.format(timestamp))
 
+    /**
+     * In the eventuality of the Date format is wrong, we create an empty [Date] object instead of throwing an exception.
```

Comment:
Proofreading
```suggestion
     * In the eventuality of the Date format being wrong, we create an empty [Date] object instead of throwing an exception.
```